wiki/markdown/subscriptions.md

```diff
@@ -12,8 +27,15 @@ comments should be a new field in the given subscription, following the json for
     }
 
 
-## Notifications
+### Types of notifications for subscriptions
 
 Today Eiffel Intelligence supports notifications by email or by sending
-a HTTP POST request.
+a HTTP POST request. It is also possible to pass parameters with the
+HTTP request. Example subscriptions with different types of notification
+can be found in the below links:
+
+* [Subscription with HTTP POST notification](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/subscription-with-REST-POST-notification.md)
```

Comment:
A very minor nitpick is that perhaps these links should be ordered the same way as in the index. REST, email and jenkins.

Reply:
Fixed, but I changed to this order in the README and site.xml instead because I think this order is better :)